### DISCUSSION TITLE
Ionic Vue & PrimeVue

### DISCUSSION THREAD

**Discussion by fotisskal** [primefaces/community#53] [PrimeVue]
I am planning to build a project using a PrimeVue dashboard template. Apart from creating the web app, I would also like to create a mobile app, both for android & ios, supporting the same exact functionalities and having the same interface. Can I combine ionic with primevue? Is it feasible and if yes, is the result satisfying without breaking components or conflict issues?

Thanks in advance.

**Comment by atkntepe** [comment]
Hi, thank you for considering PrimeVue for your project. Honestly, I don't know how it will work because, to my knowledge, we don't have an example case of using PrimeVue with Ionic. You could try starting with a small prototype to explore how well PrimeVue and Ionic can work together for your specific needs.

**Comment by cagataycivici** [comment]
We haven't any experience on our end with Ionic but if possible a started example can be added in the future. We'll work on Inertia soon and can check out Ionic as well as the new integration samples. I'll have more info by then.

**Comment by malyzkhan** [comment]
> I am planning to build a project using a PrimeVue dashboard template. Apart from creating the web app, I would also like to create a mobile app, both for android & ios, supporting the same exact functionalities and having the same interface. Can I combine ionic with primevue? Is it feasible and if yes, is the result satisfying without breaking components or conflict issues?
> 
> Thanks in advance.


Combining PrimeVue with Ionic for a unified web and mobile app with the same interface and functionality is feasible but may require adaptation of styles and layouts. It's possible to share components but expect some customization for mobile. Extensive testing is crucial to ensure a satisfying result without component conflicts.
